timezone/src/include/timezone/dialogs.rb

```diff
@@ -302,6 +302,7 @@ def SetTimeDialog
 
 
       cont = VBox(
+          VSpacing(UI.TextMode ? 1 : 0),
```

Comment:
Is it correct? Usually we add more space in graphical mode, not in text mode...

Reply:
I added this extra VSpace at the top but removed at the end in textmode removing the space between the buttons and the Header, instead of this trick which other option do you recommend?

Reply:
I'm fine with that, I just wanted to double check as usually it's the other way round.